sphinx_gallery/notebook.py

```diff
@@ -364,10 +365,22 @@ def python_to_jupyter_cli(args=None, namespace=None):
                         ' (e.g. *.py)')
     args = parser.parse_args(args, namespace)
 
+    # handle `sphinx_gallery_conf`
+    defaults = copy.deepcopy(gen_gallery.DEFAULT_GALLERY_CONF)
+    if sphinx_gallery_conf is None:
+        sphinx_gallery_conf = defaults
+    else:
+        if not isinstance(sphinx_gallery_conf, dict):
+            raise TypeError("`sphinx_gallery_conf` must be dict, "
+                            "got %s" % type(sphinx_gallery_conf))
+        for name in defaults:
+            if name not in sphinx_gallery_conf:
+                sphinx_gallery_conf[name] = defaults[name]
+
+    # run script
     for src_file in args.python_src_file:
         file_conf, blocks = split_code_and_text_blocks(src_file)
         print('Converting {0}'.format(src_file))
-        gallery_conf = copy.deepcopy(gen_gallery.DEFAULT_GALLERY_CONF)
         target_dir = os.path.dirname(src_file)
-        example_nb = jupyter_notebook(blocks, gallery_conf, target_dir)
+        example_nb = jupyter_notebook(blocks, sphinx_gallery_conf, target_dir)
```

Comment:
Corresponding revert if you want to make the suggested change via the GH UI
```suggestion
        example_nb = jupyter_notebook(blocks, gallery_conf, target_dir)
```